src/NzbDrone.Core.Test/IndexerTests/NewznabTests/NewznabRequestGeneratorFixture.cs

```diff
@@ -204,5 +204,48 @@ public void should_fallback_to_q()
             pageTier2.Url.Query.Should().NotContain("imdbid=0076759");
             pageTier2.Url.Query.Should().Contain("q=");
         }
+
+        [Test]
+        public void should_encode_raw_title()
+        {
+            _capabilities.SupportedMovieSearchParameters = new[] { "q" };
+            _capabilities.TextSearchEngine = "raw";
+
+            MovieSearchCriteria movieRawSearchCriteria = new MovieSearchCriteria
+            {
+                Movie = new Movies.Movie { Title = "Some Movie & Title: Words", Year = 2021, TmdbId = 123 },
+                SceneTitles = new List<string> { "Some Movie & Title: Words" }
+            };
+
+            var results = Subject.GetSearchRequests(movieRawSearchCriteria);
+
+            var page = results.GetTier(0).First().First();
+
+            page.Url.Query.Should().Contain("q=");
+            page.Url.Query.Should().NotContain(" & ");
```

Comment:
I think this should test for the exact string we expect to get back, not some very tiny partial matches, since it's URI encoded this will never be true.

Reply:
makes sense; pushed